lib/go/edgecontext/edgecontext.go

```diff
@@ -20,9 +21,19 @@ import (
 // LoIDPrefix is the prefix for all LoIDs.
 const LoIDPrefix = "t2_"
 
-// ErrLoIDWrongPrefix is an error could be returned by New() when passed in LoID
-// does not have the correct prefix.
-var ErrLoIDWrongPrefix = errors.New("edgecontext: loid should have " + LoIDPrefix + " prefix")
+// Locale codes should contain either a language, or a language and region specifier
+// separated by an underscore.
+// e.g. en, en_US
+var LocaleRegex = regexp.MustCompile(`^[a-z]{2}(_[A-Z]{2})?$`)
```

Comment:
See the complain from golint: https://github.com/reddit/edgecontext/pull/12/checks?check_run_id=3118847071:

```
comment on exported var LocaleRegex should be of the form "LocaleRegex ..."
```

Reply:
Huh, looks like my local golint is having some issues. Fixed in https://github.com/reddit/edgecontext/pull/12/commits/2b33a68fe822766da8087329498d557104f4a9a6